packages/cli/package.json

```diff
@@ -57,10 +52,7 @@
     "semver": "^5.0.3",
     "serve-static": "^1.13.1",
     "shell-quote": "1.6.1",
-    "slash": "^2.0.0",
-    "ws": "^1.1.0",
-    "xcode": "^2.0.0",
-    "xmldoc": "^0.4.0"
+    "ws": "^1.1.0"
```

Comment:
would be good to update it, latest is v6 :o

Reply:
Yeah, but this is proxy stuff, so I am a bit worried of doing it here.